app/src/main/admin/components/Touchstones/SingleTouchstoneVersion/ResponsibilitiesPage.tsx

```diff
@@ -8,13 +8,9 @@ import {touchstoneResponsibilitiesPageActionCreators} from "../../../actions/pag
 import {ResponsibilityList} from "./ResponsibilityList";
 import {ResponsibilitySetWithExpectations} from "../../../../shared/models/Generated";
 import {compose} from "recompose";
+import {TouchstoneVersionPageLocationProps} from "./TouchstoneVersionPage";
 
-export interface ResponsibilitiesPageLocationProps {
```

Comment:
This is exactly the same as `TouchstoneVersionPageLocationProps` so I thought we could just share that between components.